docs/source/nlp/usage_guides/nlp_properties_extended.rst

```diff
@@ -86,3 +130,14 @@ The property is calculated by identifying key entities and quantities in the LLM
 and prices, and then identifying the same entities and quantities in the input given to the LLM.
 The property is calculated as the ratio of the number of entities/quantities in the LLM output that are also in the
 input, divided by the total number of entities/quantities in the LLM output.
+
+Examples
+~~~~~~~~
+
+======================================================================================================================  =====================================================  ===================
+LLM Input                                                                                                               LLM Output                                             Grounded in Context
+======================================================================================================================  =====================================================  ===================
+Michael Jordan (1963) is an American former professional basketball player and businessman. In what year was he born?   He was born in 1963.                                   1.0
+Michael Jordan (1963) is an American former professional basketball player and businessman. In what year was he born?   Michael Jeffrey Jordan was born in 1963                0.5
+Michael Jordan (1963) is an American former professional basketball player and businessman. In what year was he born?   He won many NBA championships with the Chicago Bulls   0.0
```

Comment:
This example is misleading IMO... shouldn't we demonstrate with a different question everytime?
e.g. "In which group did he play?"
Cause the from the example it seems that it's not answering the question, more than it's just using other information not from the context

Reply:
right, changed the last question